composer.json

```diff
@@ -3,12 +3,12 @@
     "description": "PHP CS Fixer config for Paysera conventions",
     "type": "library",
     "require": {
-        "php": ">=7.4",
+        "php": "^7.4 || ^8.1",
```

Comment:
If we want to avoid PHP 8.0 something like that would be better `>=7.4 <8.0 || >=8.1`.

Reply:
I made requirements like for php-cs-fixer (php: ^7.4 || ^8.0) but with a skip of 8.1. We can't guarantee that it will work with PHP 9.*, so I believe that ^8.1 is better than >=8.1.
But if you insist on changing it - no problem. I will change it. Just mention that once again in a comment.

Reply:
Well, we also cannot guarantee that it won't work with PHP 9 :) In general, it's common practice to set lowest supported version of platform, eg.
- https://github.com/symfony/symfony/blob/3.4/composer.json#L19
- https://github.com/symfony/symfony/blob/4.4/composer.json#L35
- https://github.com/symfony/symfony/blob/5.4/composer.json#L35

It's because developers are usually making backwards-incompatible changes only when they have to, and even then, in case of PHP, only some functions become backwards-incompatible, not whole language. Moreover, it's way better to not limit max supported version, because in case everything works fine there will be no need to change anything. And in case everything works fine and we limit max version then we'd need to make config change... It's better to fix issue in the code if something does not work instead of changing version when this could have been avoided.

So lets not limit max version.

Reply:
Fixed